AppCenterCrashes/AppCenterCrashesTests/MSACErrorLogFormatterTests.mm

```diff
@@ -766,97 +764,41 @@ - (void)checkIfArchNameConvertedCorrectly:(NSNumber *)codeType archName:(NSNumbe
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM64) expectedType:@"ARM-64"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86) expectedType:@"X86"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86_64) expectedType:@"X86-64"];
+  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_POWERPC) expectedType:@"PPC"];
 }
 
 - (void)testConvertArchNameToString {
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V6)
-                  isKnownEncodingType:YES
                          expectedName:@"armv6"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V7)
-                  isKnownEncodingType:YES
                          expectedName:@"armv7"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM)
                              archName:@(CPU_SUBTYPE_ARM_V7S)
-                  isKnownEncodingType:YES
                          expectedName:@"armv7s"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64_ALL)
-                  isKnownEncodingType:YES
                          expectedName:@"arm64"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64_V8)
-                  isKnownEncodingType:YES
                          expectedName:@"armv8"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_ARM64)
                              archName:@(CPU_SUBTYPE_ARM64E)
-                  isKnownEncodingType:YES
                          expectedName:@"arm64e"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_X86)
                              archName:@(CPU_TYPE_X86)
-                  isKnownEncodingType:YES
                          expectedName:@"i386"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_X86_64)
                              archName:@(CPU_TYPE_X86_64)
-                  isKnownEncodingType:YES
                          expectedName:@"x86_64"];
   [self checkIfArchNameConvertedCorrectly:@(CPU_TYPE_POWERPC)
                              archName:@(CPU_TYPE_POWERPC)
-                  isKnownEncodingType:YES
                          expectedName:@"powerpc"];
 }
 
-- (void)testConvertCodeTypeUnknownEncodingTypeToString {
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM) isKnownEncodingType:NO expectedType:@"???"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_ARM64) isKnownEncodingType:NO expectedType:@"???"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86) isKnownEncodingType:NO expectedType:@"???"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_X86_64) isKnownEncodingType:NO expectedType:@"???"];
-  [self checkIfCodeTypeConvertedCorrectly:@(CPU_TYPE_POWERPC) isKnownEncodingType:NO expectedType:@"???"];
-}
-
-- (void)testConvertArchNameUnknownEncodingTypeToString {
```

Comment:
It is still possible that archName will be set to "???" if `archName` (first parameter in `convertArchNameToString`) was not matched to the list in `switch`. Shouldn't we cover this scenario?